README.md

```diff
@@ -33,8 +36,14 @@ Our splitting expression is "users". As the output of the component we'll have t
     "name": "Mike"
 }
 ```
+*Notes:*
+
+- *When splitting expression refers to an object splitter returns this object;*
+- *When splitting expression contains primitive value like ```users:"John"``` or array of primitives like ```users:["John", "Mike", "Anna"]``` splitter emits error.*
+
+#### List of Expected Config fields
+```Split Property``` - use this field to choose a separator.
 
-If splitting expression refers to object splitter just return this object.
+## Known limitations (common for the component)
+No.
 
```

Comment:
## Documentation links
More information and some examples can be found here: [Splitter documentation](https://www.elastic.io/connectors/splitter-integration/)
and here: [Array splitting and JSONata mapper](https://support.elastic.io/support/solutions/articles/14000069604-array-splitting-and-jsonata-mapper)